README.md

```diff
@@ -1,6 +1,14 @@
 # django-garnett
 
-Django Garnett is a field level translation library that allows you to store strings in multiple languages for fields in Django - with minimal changes to your models and without having to rewrite your code (mostly).
+Django Garnett is a field level translation library that allows you to store strings in multiple languages for fields in Django - with minimal changes to your models and without having to rewrite your code.
+
+Want a demo? https://django-garnett.herokuapp.com/
```

Comment:
How much is this review app costing us? Might be good to look at a longer term solution to this

Reply:
$0.

Reply:
Its on a personal free tier app.